.github/workflows/regression-tests.yml

```diff
@@ -31,7 +33,11 @@ jobs:
           cat /proc/cpuinfo
           ulimit -a
           env
+
       - name: Configure & Build
+        # We do not run debug epoll
+        if: ${{ matrix.proactor != 'Epoll' || matrix.build-type == 'Release' }}
```

Comment:
gh run https://github.com/dragonflydb/dragonfly/actions/runs/12668947653/job/35305396327 

Admittedly we now spawn 4 more jobs:

2 for `Debug(x86 && arm) + epoll`
2 for `Release(x86 && arm) + epoll`  

The 2 `Debug` will `complete immediately` because we skip them with the `if condition above`. 

It would be nice if we didn't get the noise of the two jobs but the `if` conditions apply only `after matrix` is chosen, so we can't really skip a job based on the matrix configuration (or at least that's how I understand it from the docs).

Reply:
Small update I found `exclude` -- I will try this

Reply:
Worked like charm :tada: 

See https://github.com/dragonflydb/dragonfly/actions/runs/12669075440